client/js/components/App.react.jsx

```diff
@@ -53,17 +118,33 @@ export default class App extends React.Component {
                     trailGeo={this.state.trailGeo} />
             );
         }
-        else {
+        else if (this.state.allParks && this.state.allParksTopo) {
+            var parkFilters;
+            if (this.state.amenityLookup && this.state.facilityLookup) {
+                parkFilters = <ParkFilters
+                    amenityLookup={this.state.amenityLookup}
+                    facilityLookup={this.state.facilityLookup}
+                    setFilter={(filter) => this.applyFilters(filter)} />
+            }
             content = (
-                <ParkList
-                    parks={this.state.allParks}
-                    onSelectPark={(park) => this.selectPark(park)} />
+                <div>
+                    {parkFilters}
+                    <ParksMap
+                        visibleParkIds={this.state.visibleParkIds}
+                        parksTopo={this.state.allParksTopo}
+                        onSelectPark={(parkId) => this.selectParkWithId(parkId)} />
+                    <ParksList
+                        parks={this.state.visibleParks}
+                        onSelectPark={(park) => this.selectPark(park)} />
+                </div>
             );
         }
         return (
             <div>
-              <Navigation />
-              <div className='container'>{content}</div>
+              <div className='container'>
+                  <Navigation />
```

Comment:
since we have the 'container' div wrapping everything, im thinking the outer div could go away